tests/test_server.py

```diff
@@ -18,11 +19,14 @@ def rtm_start_fixture():
     return json_login_data
 
 
-def test_server(server):
+def test_server():
+    server = Server("valid_token", connect=False, )
     assert type(server) == Server
+    assert server == "valid_token"
```

Comment:
the server is coerced into a string? is that a useful feature?

Reply:
No idea. It's a Preexisting Feature™

I've never used it, but I'm sure someone has at some point during the history of this project.